components/Partners/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./partners.module.css";
+
+const Partners: React.FC = () => {
+  return (
+    <section className={styles["container"]}>
+      <div className={styles["partner"]}>
+        <img
+          src="https://www.stevefund.org/wp-content/uploads/2019/03/The-Steve-Fund-logo-vertical-color.png"
+          alt="Partner logo"
```

Comment:
Make sure the alt attributes for images directly describe the image next time so that screen readers can describe the image